docs/topics/modules/planning_development_work.adoc

```diff
@@ -1,54 +1,29 @@
-
-. Create the following child *Task* work items to implement the *Feature* using the in-line add option:
-
-* <<creating_new_project-hello-world,*Launch a sample quickstart application*>>
-* <<approving_build_pipeline,*Test the application in Stage and approve to the Run environment*>>
-* <<creating_che_workspace-hello-world,*Modify the quickstart codebase*>>
-* <<reviewing_publishing_changes-hello-world,*Review and publish changes to your codebase*>>
+. Using the in-line add option create an execution oriented child *Story* for the *Epic* with the title *User should be able to easily launch a sample Vert.x application* and a child *Task* for the story titled *Review and publish changes to your codebase*.
 +
-image::gs_tasks_egs.png[Example Tasks]
+See link:user-guide.html#working_with_work_items[working with work items] for more information about modifying work items.
 
-. Similarly, select *Fundamental* in the *Add Work Item* pane, to create a fundamental work item with the title *The application should be secure with appropriate licenses* and use the in-line add option to create a child experience, feature, and task for it. The task can be titled <<analytics_stack_report,*Use analytics to identify and resolve security and license issues in the application*>>.
-+
-image::gs_fundamental_task_eg.png[Example Fundamental Task]
-+
-. In the left panel, click the (+) icon adjacent to *Iterations* to link:user-guide.html#creating_a_new_iteration[create an iteration] called *Test Iteration* and force activate it.
-. Click on each of the feature and child task work items to preview it, and:
-
-* link:user-guide.html#tracking_state_of_a_work_item[Change the state of the work items] to *In Progress* when you start working on them
-*  link:user-guide.html#assigning_the_work_item[Add yourself as the assignee]
-* link:user-guide.html#associating_work_items_with_an_iteration-user-guide_iterations[Associate the work items to the *Test Iteration*]
-+
-See link:user-guide.html#working_with_work_items[working with work items] for detailed information about modifying and tracking work items by associating them to iterative development cycles.
+. In the left panel, click the (+) icon adjacent to *Iterations* to link:user-guide.html#creating_a_new_iteration[create an iteration] called *Test Iteration*, force activate it, and  link:user-guide.html#associating_work_items_with_an_iteration-user-guide_iterations[associate the story and task to the *Test Iteration*].
 +
-image::gs_modify_wi.png[Modify work item]
+image::gs_associate_wi_with_iteration.png[Associate work item with an iteration]
 
-You can now click *Test Iteration* to see the work items associated with it and use the planner to track your development work for the iteration.
+You can now use the *Test Iteration* in the left panel to see the work items associated with it and track your work.
```

Comment:
One thing you probably already have on your radar for later, but we should have some instructions later when we do something about returning to the planner and updating the relevant work items.

Reply:
This was already in place for the earlier version, what Inow need to do is depending on how this module goes modify or delete existing stuff.